CODEOWNERS

```diff
@@ -176,6 +176,8 @@
 /drivers/i2c/i2c_rv32m1_lpi2c*            @henrikbrixandersen
 /drivers/i2c/*sam0*                       @Sizurka
 /drivers/i2c/i2c_dw*                      @gnuless
+/drivers/i2s/Kconfig.nrfx                 @adamkondraciuk
```

Comment:
Is there some example code you can add as well? Having a sample that e.g. sends incremental numbers over I2S would greatly assist developers in getting started.

Reply:
> Is there some example code you can add as well? Having a sample that e.g. sends incremental numbers over I2S would greatly assist developers in getting started.

I'll push it as another PR after driver review